client/app/layouts/splash.html

```diff
@@ -1,4 +1,18 @@
-<language-chooser></language-chooser>
+<div>
+  <language-chooser></language-chooser>
+  <md-button href="https://github.com/yunity/foodsaving-frontend">
+    <i class="fa fa-github"></i>
+    Github
+    <md-tooltip md-direction="bottom">
```

Comment:
Also possible! I was using this before, but I found that using one common method would be better.
The translate tag wasn't useful in some cases, so I would prefer to only use the translate attribute.